### DISCUSSION TITLE
Window Scroll Event

### DISCUSSION THREAD

**Discussion by antletem** [anvilistas/anvil-extras#417] [Q&A]
I am not sure if this should be an issue or not. 

When adding routing to an application of mine, the window scroll event no longer fires.

Let me add some code to the words.

I have the following form:

```python
from ._anvil_designer import Form2Template
from anvil import *
from anvil.js import window
from anvil_extras import routing

@routing.route('')
class Form2(Form2Template):
  def __init__(self, **properties):
    # Set Form properties and Data Bindings.
    self.init_components(**properties)

    # Any code you write here will run before the form opens.
    window.addEventListener('scroll',self.scroll_event)

  def scroll_event(self,*args,**event_args):
    print("scrolling")
```

prior to adding the routing, the scroll event would fire. Now it no longer does.

Here is the clone:

https://anvil.works/build#clone:S3GSBNOJB3D5YEO4=CKB6SKESWPHVFSYSMPQXSDY2

Am I missing some abstraction that happens with routing? I tried using the document object as well. This seems to happen with any event I try and add (I say any, but I only tried ```visibilitychange``` as well).

Anyone have any insight on what could be going on?

**Comment by s-cork** [comment]
What behaviour am I supposed to be noticing in the clone?
All I see is a bunch of print statements whenever the page is scrolled.

Could you give exact steps for the issue and how to reproduce it?

**Comment by antletem** [comment]
It must be a device issue. I'm working remotely today, and it is working fine on my home PC. At work I have a windows 10 2020 optiplex. Same browser, but my home browser was updated last night, so maybe that plays a role as well...

Either way, thank you for taking a look. I knew this shouldn't be an issue.
